x-pack/platform/plugins/shared/alerting/common/routes/rule/apis/global_execution_summary/types/v1.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { TypeOf } from '@kbn/config-schema';
+import type {
+  getGlobalExecutionSummarySchema,
```

Comment:
This is a small thing, but I think these should be `getGlobalExecutionSummarySchemaV1` and `getGlobalExecutionSummaryResponseBodySchemaV1 `.

Reply:
I wrote it like that because all the other endpoints I checked in that folder follow the same pattern: they don't put the v1 where the types are declared and they do `export ... as ` in the index file. So I would rather keep it like that for consistency 🙏

Reply:
In the `v1.ts` files they import them as schemaV1 for example,` x-pack/platform/plugins/shared/alerting/common/routes/rule/apis/create/types/v1.ts`. It's not a big issue, but all of the endpoints do follow that pattern